src/Models/SyncMapping.php

```
@@ -121,6 +121,20 @@ public static function findByExternalKey(Database $db, int $integrationId, strin
         return $row !== false ? $row : null;
     }
 
+    public static function findWithIntegrationByExternalKey(Database $db, string $externalKey): ?array
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)